README.md

```diff
@@ -336,10 +336,13 @@ Statics parameters
 - `cssPath`: Subdirectory within `staticsRoot` where your CSS files are located. By default this folder will not be made public, but is instead meant to store unminified CSS source files which will be minified and stored elsewhere when the app is started.
   - Default: `css`
 - `cssCompiler`: Which CSS preprocessor, if any, to use. Must also be marked as a dependency in your app's package.json. Set to `none` to use no CSS preprocessor.
-  - Default: `{nodeModule: 'roosevelt-less', params: {compress: true}}`.
+  - Default: `{nodeModule: 'roosevelt-less', params: {cleanCSS: {advanced: true, aggressiveMerging: true}, sourceMap: {}}}`.
```

Comment:
Might have to tweak this default, see relevant comment https://github.com/rooseveltframework/roosevelt-less/pull/22#pullrequestreview-76784533